html-lists/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Lists</title>
+</head>
+<body>
+<h1>HTML Lists</h1>
+<h2>Ordered List Example</h2>
+<ol>
+  <li>Learn to code.</li>
+  <li>Build projects.</li>
+  <li>Get a job.</li>
+</ol>
+<h2>Unordered List Example</h2>
+<ul>
+  <li>HTML</li>
+  <li>CSS</li>
+  <li>JS</li>
+  <li>SQL</li>
+  <li>PHP</li>
+</ul>
```

Comment:
These should all be indented by one tab, to make it clear that they are children of the `body` element